.azure-pipelines/azure-pipelines-osx.yml

```
@@ -35,15 +35,15 @@ jobs:
 
   - script: |
       source activate base
-      conda install -n base -c conda-forge --quiet --yes conda-forge-ci-setup=2 conda-build shyaml
+      conda install -n base -c conda-forge --quiet --yes conda-forge-ci-setup=2 conda-build
     displayName: 'Add conda-forge-ci-setup=2'
 
   - script: |
       source activate base
       echo "Configuring conda."
 
       setup_conda_rc ./ ./recipe ./.ci_support/${CONFIG}.yaml
-
+      export CI=azure
```

```suggestion
      export CI=azure 
```
trigger build please

---

package is finally available, it didn't build because of some dependency that was being built at the time.
https://anaconda.org/conda-forge/mesa-libgl-devel-cos7-aarch64